spec/Section 3 -- Type System.md

```diff
@@ -1360,11 +1360,57 @@ Literal Value            | Variables               | Coerced Value
 
 **Type Validation**
 
-1. An Input Object type must define one or more input fields.
-2. The fields of an Input Object type must have unique names within that
-   Input Object type; no two fields may share the same name.
-3. The return types of each defined field must be an Input type.
+* An Input Object type must define one or more input fields.
```

Comment:
It should be consistent with other sections `Type Validation` that have numbered lists.